### ISSUE TITLE
nvda 2026.1: unable to load sapi4 synthesizer

### ISSUE THREAD

**Issue by acerbt** [nvaccess/nvda#20302]
### Brief summary

If I currently try to load the sapi4 synth in nvda 2026.1, the synth fails to load. This appears to line up with something about lack of pitch handling on the sapi4 voice, since the sapi4 voice that loads when nvda did properly load the synth was a rare example of one that doesn't support rate and pitch control. This issue did not happen in any nvda 2025 releases, and only started in 2026 ones.

### Steps to reproduce

1. Start NVDA
2. enter the synthesizer dialog (nvda ctrl s)
3. if installed, attempt to select sapi4

### Actual behavior

if the sapi4 voice that is selected by default as the driver is loaded has no pitch control, the loading of the synth will fail and the fall back logic runs, returning to the previously loaded synth driver. I'm unsure what happens if you can load it successfully, then switch to a voice without pitch control.
NVDA+ctrl+s
Select Synthesizer  dialog
Synthesizer:  combo box  Microsoft Speech API version 5 (64 bit)  collapsed  Alt+  s
up arrow
Microsoft Speech API version 5 (32 bit)
up arrow
Microsoft Speech API version 4
enter
Synthesizer Error  dialog  Could not load the sapi4_32 synthesizer.
OK  button
pressed
Select Synthesizer  dialog
Synthesizer:  combo box  Microsoft Speech API version 4  collapsed  Alt+  s
escape

### Expected behavior

all properly configured sapi4 voices should be available, and the synth driver should not fail to load.

### NVDA logs, crash dumps and other attachments

[nvda with sapi4 synth change attempt.log](https://github.com/user-attachments/files/28736587/nvda.with.sapi4.synth.change.attempt.log)

### NVDA type

installed copy

### NVDA version

2026.1 and 2026.1.1 (2026.1.1.55980)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

earlier versions including the last major versions of nvda 2025 do properly load the synth driver with the same voice selected on sapi4.

### Windows version

Windows 11 25H2 (AMD64) build 26200.8457

### Name and version of other software in use when reproducing the issue

loquendo tts engine version 7 (sapi4 synth installed when the error happens). Voices from this engine in sapi4 happen not to support rate and pitch, even though the sapi5 ones do.

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have run the System Accessibility Repair Tool and the issue still occurs

**Comment by michaelDCurran** [comment]
I have tried to fix this in pr #20432. Please test this [try build](https://download.nvaccess.org/snapshots/try/try-i20302/nvda_snapshot_try-i20302-56756,da9e2d53.exe) with the sapi4 synth that does not support pitch / rate and let me know if  it works well now.

**Comment by acerbt** [comment]
> I have tried to fix this in pr [#20432](https://github.com/nvaccess/nvda/pull/20432). Please test this [try build](https://download.nvaccess.org/snapshots/try/try-i20302/nvda_snapshot_try-i20302-56756,da9e2d53.exe) with the sapi4 synth that does not support pitch / rate and let me know if it works well now.

Loading the sapi 4 synth works in this one. Previously it wouldn't load at all sicne the voice I had chosen was one which didn't support rate and pitch but it works normally in the test build.
